package.json

```diff
@@ -46,4 +46,4 @@
 	"peerDependencies": {
 		"@nestjs/common": "^9.0.0"
```

Comment:
We have to change "peerDependencies" too, in order to work.

Reply:
Sorry, I hadn't realised they were using peerDependencies. Fixed.